lib/solr_wrapper.rb

```diff
@@ -10,6 +10,10 @@ def self.default_instance_options
     @default_instance_options ||= {}
   end
 
+  def self.default_instance_options=(options)
```

Comment:
@jcoyne good thought, but since SolrWrapper is a module that is undefined.  Can anyone point to where the module is included?  Should I think about an ActiveSupport included do block?